collect.py

```diff
@@ -203,6 +203,18 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    source = driver.page_source
```

Comment:
Is this equivalent to `document.documentElement.outerHTML`?

Reply:
yes it does

Reply:
Are you 100% sure? Can you show me the documentation where you found that they are equivalent?

Reply:
here are a few websites including the official selenium website, that dont technically compare them both but states the function of this method-

1. https://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html?highlight=page_source#selenium.webdriver.remote.webdriver.WebDriver.page_source
2.  https://stackoverflow.com/questions/7861775/python-selenium-accessing-html-source

i also ran it on my local machine and here are the results- 
![2018-03-22](https://user-images.githubusercontent.com/22891797/37763557-71fc65ac-2de5-11e8-8ce9-2331a4b8c723.png)

Reply:
So, document.documentElement.outerHTML doesn't get the source of the page, but the DOM tree.
The documentation you shared says "page_source" returns the source of the page.
As soon as you open a page, like you did in your example, they might be equivalent. But what if some JavaScript script on the page changes something in the DOM, or if the user writes something in a text field, etc.? Are they still the same in this case or does page_source return just the original source of the page?

Reply:
@marco-c  you're right, my bad, the page source wont change with javascript
https://stackoverflow.com/questions/17333582/selenium-do-javascript-actions-reflect-in-driver-page-source

i fixed it, check it